srcpkgs/odin/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'odin'
+pkgname=odin
+version=0.10.0
+revision=1
+archs="x86_64*"
+wrksrc="Odin-${version}"
```

Comment:
I think convention in Void is to make the package name match the case, so `Odin`

Reply:
Can we get more opinions on this? Back in the day we kinda went either way. 

I've always disliked starting a package name with a capital since xbps is case sensitive which makes packages harder to deal with. Also, the executable itself is lower case, I've always found this to just bring in more confusion.

If that's what we do now, that's fine by me (I prefer consistency either way) , but I'd like more opinions before I go ahead and make the change.